### ISSUE TITLE
There are tables on the website that do not have titles

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#171]
Screengrab/videos (insert file names here)
2020 09 07, Table has no Name attribute site 1.mp4
[2020 09 07, Table has no Name attribute site 1.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5183527/2020.09.07.Table.has.no.Name.attribute.site.1.mp4.zip)

2020 09 07, Table has no Name attribute site 2.mp4
[2020 09 07, Table has no Name attribute site 2.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5183556/2020.09.07.Table.has.no.Name.attribute.site.2.mp4.zip)

2020 09 07, Table has no Name attribute site 3.mp4
[2020 09 07, Table has no Name attribute site 3.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5183560/2020.09.07.Table.has.no.Name.attribute.site.3.mp4.zip)

2020 09 07, Table has no Name attribute site 4.mp4
[2020 09 07, Table has no Name attribute site 4.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5183561/2020.09.07.Table.has.no.Name.attribute.site.4.mp4.zip)

Link
1.	https://d8-stg.boston.gov/calendar/mayor-main-trolley-tour-day-3
2.	https://d8-stg.boston.gov/departments/digital-team/test-page
3.	https://d8-stg.boston.gov/calendar/mayor-main-trolley-tour-day-1
4.	https://d8-stg.boston.gov/departments/emergency-medical-services/how-apply-ems-special-event-permit

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 85

Trusted Tester Test
Tables

Brief Description
The table on the website does not have a title

Steps to reproduce
This is for 2020 09 07, Table has no Name attribute site 1.mp4
1.	Go to the site: https://d8-stg.boston.gov/calendar/mayor-main-trolley-tour-day-3
2.	00:05 – Slide down the page to the table labeled "2019 Winners"
3.	00:12 – Bug: The table has no name attribute, when it should have one
o	Note:  the table cells do contain accessibility information.

This is for 2020 09 07, Table has no Name attribute site 2.mp4
1.	Go to the site: https://d8-stg.boston.gov/departments/digital-team/test-page 
2.	00:03 – Slide down the page to the table labeled “Commission Members”
3.	00:06 – Bug: The table has no name attribute, when it should have one
o	Note:  the table cells do contain accessibility information.

This is for 2020 09 07, Table has no Name attribute site 3.mp4
1.	Go to the site: https://d8-stg.boston.gov/calendar/mayor-main-trolley-tour-day-1 
2.	00:03 – Slide down the page to the table labeled "2019 Winners"
3.	00:09 – Bug: The table has no name attribute, when it should have one
o	Note:  the table cells do contain accessibility information.

This is for 2020 09 07, Table has no Name attribute site 4.mp4
1.	Go to the site: https://d8-stg.boston.gov/departments/emergency-medical-services/how-apply-ems-special-event-permit 
2.	00:03 – Slide down the page to the table labeled "Boston EMS Fees"
3.	00:10 – Bug: The table has no name attribute, when it should have one
o	Note:  the table cells do contain accessibility information.

Desired Result
•	All of the tables on the various pages mentioned above have titles

Actual Result
•	None of the tables on the pages mentioned above have titles.
o	This is described as being necessary in the link: https://www.w3.org/WAI/tutorials/tables/caption-summary/

**Comment by david-iterators** [comment]
Note that this current bug clarifies and replaces https://github.com/CityOfBoston/Iterators/issues/132.

The tables are missing the title/caption attribute to help with navigation.  Note that the actual cells do contain the needed accessibility information.

Additional information for the titles/captions can be found at:  https://www.w3.org/WAI/tutorials/tables/caption-summary/

**Comment by subaha-cob** [comment]
@andrebtlima Most of the table content was written directly on the page. I will make the fix in UAT, but after that I will be making the fix on Production directly. No need to make it on Dev and Staging. Once the other two environment downloads the latest database from Production they will also get the changes.

Also we need to finish writing the documentation to hand off to content editors so they are aware of this little but very important accessibility issue.
